src/preferences/dialog/dlgpreferences.cpp

```diff
@@ -65,6 +65,14 @@ DlgPreferences::DlgPreferences(
     setupUi(this);
     contentsTreeWidget->setHeaderHidden(true);
 
+    // Add '&' to default button labels to always have Alt shortcuts, indpependent
+    // of operating system.
+    buttonBox->button(QDialogButtonBox::Help)->setText(tr("&Help"));
```

Comment:
This is a translators trap, because assigning the short cuts without conflicts is now part of his task. So we need to inform them that they need to consider all the strings below at once. This can be done via:
```
//: ...
```  
Idea: How about translate these strings in a common string and split them up at a `;` or such?

Reply:
Ah okay, didn't think of that.
Never seen a common tr that's split up again, though. Would mentioning the other buttons as tr context hint suffice?

Reply:
Yes